test/evergreen/code_change_report/code_change_report.py

```diff
@@ -159,8 +268,133 @@ def generate_file_info_as_html_text(file: str, file_info: dict, verbose: bool):
     return report
 
 
+# change_string generates an html string indicating a change (if any) between two values
+def change_string(old_value: int, new_value: int) -> str:
+    result = ""
+    if new_value > old_value:
+        # up arrow
+        result = "&#8679;{}".format(new_value - old_value)
+    elif new_value < old_value:
+        # down arrow
+        result = "&#8681;{}".format(old_value - new_value)
+
+    return result
+
+
+# describe_complexity_categories generates an html table describing the complexity categories
+def describe_complexity_categories() -> list:
+    code_colour_ = get_complexity_html_colour(1)
+    description = list()
+    description.append("<table class=\"center\">\n")
+    description.append("<tr>\n")
+    description.append(
+        "<th><a href='https://en.wikipedia.org/wiki/Cyclomatic_complexity'>Cyclomatic complexity</a></th></th>\n")
+    description.append(
+        "<th><a href='https://en.wikipedia.org/wiki/Cyclomatic_complexity#Interpretation'>Risk evaluation</a></th>\n")
+    description.append("</tr>\n")
+    description.append("<tr><td> {} </td><td> Simple procedure, little risk </td></tr>\n".
+                       format(value_as_centred_text(get_complexity_html_colour(1), "1-10")))
+    description.append("<tr><td> {} </td><td> More complex, moderate risk   </td></tr>\n".
+                       format(value_as_centred_text(get_complexity_html_colour(11), "11-20")))
+    description.append("<tr><td> {} </td><td> Complex, high risk            </td></tr>\n".
+                       format(value_as_centred_text(get_complexity_html_colour(21), "21-50")))
+    description.append("<tr><td> {} </td><td> Untestable code, very high risk    </td></tr>\n".
+                       format(value_as_centred_text(get_complexity_html_colour(51), ">50")))
+    description.append("</table>\n")
+    return description
+
+
+# coverage_string converts a coverage value ana a total into a colour-coded html string
```

Comment:
```suggestion
# coverage_string converts a coverage value and a total into a colour-coded html string
```

Reply:
(Technically `colour` should also become `color`, but we're not running s_all on these scripts so I'm happy to leave as is)